compliance-report/generate.sh

```diff
@@ -2,6 +2,23 @@
 
 set -eux
 
+# Get release creator.
+cd $GITHUB_WORKSPACE
+export GH_TOKEN=${GH_TOKEN:-TOKEN}
```

Comment:
How about handling this the same way as in authorized-pub?
```suggestion
if [ -n "$TOKEN" ]; then
  export GH_TOKEN=$TOKEN
fi
```

Either way, if `GH_TOKEN` isn't already set, the new token will always be `TOKEN`, instead of the value of `$TOKEN`.